components/analytics/system_analytics.jsx

```diff
@@ -120,13 +115,8 @@ export default class SystemAnalytics extends React.Component {
                             />
                         }
                         data={userCountsWithPostsDay}
-                        options={{
-                            legend: {
-                                display: false
-                            }
-                        }}
```

Comment:
Removing this doesn't result in a behavior or UI change?

Reply:
No, I moved the chartOptions inside the component, because every usage of the component passes the same option. Probably I would do the same with the data field, changing it to be more "semantic" to the component, and less Chart.js dependant, I mean, encapsulating better the concept of line chart. Currently the rest of the app must know how to pass data to Chart.js to pass data to the components.

Anyway, that is other conversation. The short answer is, I moved it to the component.